command/debug.go

```diff
@@ -994,27 +981,29 @@ func (c *DebugCommand) captureError(target string, err error) {
 	c.errLock.Unlock()
 }
 
-func (c *DebugCommand) writeLogs(ctx context.Context) error {
+func (c *DebugCommand) writeLogs(ctx context.Context) {
 	out, err := os.Create(filepath.Join(c.flagOutput, "vault.log"))
 	if err != nil {
-		return err
+		c.captureError("log", err)
+		return
 	}
-	defer out.Close()
```

Comment:
Why is this being removed?

Reply:
Sorry I missed this.  I think it's a mistake, I'll restore it.